test/flutter_graphql_utils_test.dart

```diff
@@ -1,9 +1,9 @@
 import 'package:flutter/widgets.dart';
-import 'package:flutter_graphql_client/src/sil_graphql_utils.dart';
+import 'package:flutter_graphql_client/src/flutter_graphql_utils.dart';
 import 'package:test/test.dart';
 
-import './test_utils.dart';
 import 'mocks.dart';
+import 'test_utils.dart';
 
 void main() {
   group('SILGraphQlUtils', () {
```

Comment:
?? did you mean FlutterGraphQlUtils

Reply:
the import for FlutterGraphQlUtils is still present, this is for `generateMockGraphQLClient` method